crypto/src/ed25519.rs

```diff
@@ -223,8 +242,36 @@ impl Display for Ed25519Signature {
 
 impl Default for Ed25519Signature {
     fn default() -> Self {
-        let sig = <ed25519_dalek::Signature as Signature>::from_bytes(&[0u8; 64]).unwrap();
-        Ed25519Signature(sig)
+        <Ed25519Signature as Signature>::from_bytes(&[1u8; ED25519_SIGNATURE_LENGTH]).unwrap()
+    }
+}
+
+impl Serialize for Ed25519Signature {
```

Comment:
as discussed serde_as would produce `{"sig": base64_encoded_string}` instead of a flattened `base64_encoded_string`, removed related functions in serder_helper